tsfile/src/main/java/org/apache/iotdb/tsfile/fileSystem/FSFactoryProducer.java

```diff
@@ -33,36 +33,29 @@
 public class FSFactoryProducer {
 
   private static FSType fSType = TSFileDescriptor.getInstance().getConfig().getTSFileStorageFs();
-  private static HDFSFactory hdfsFactory = new HDFSFactory();
-  private static LocalFSFactory localFSFactory = new LocalFSFactory();
-  private static HDFSInputFactory hdfsInputFactory = new HDFSInputFactory();
-  private static LocalFSInputFactory localFSInputFactory = new LocalFSInputFactory();
-  private static HDFSOutputFactory hdfsOutputFactory = new HDFSOutputFactory();
-  private static LocalFSOutputFactory localFSOutputFactory = new LocalFSOutputFactory();
-
```

Comment:
```
  private static FSFactory fsFactory;
  private static FSInputFactory fsInputFactory;
  private static FSOutputFactory fsOutputFactory;
  
  static {
        if (fSType.equals(FSType.HDFS)) {
              // init hdfs factories
        } else {
             // init local factories
        }
  }
```
This may avoid calling too many new(), and hopefully, when not using HDFS you won't get any HDFS log.